vip-dashboard/components/widget-contact.php

```diff
@@ -46,8 +46,8 @@ function render_vip_dashboard_widget_contact() {
 				<div class="contact-form__input">
 					<textarea name="details" rows="8" id="contact-form__details" placeholder="Please include as much information possible about the issue, including relevant domain names and your investigation so far."></textarea>
 					<strong>
-						Note: please read our <a href="https://docs.wpvip.com/documentation/general-ticket-guidelines/" target="_blank" rel="noopener">full ticket guidelines</a>
-						for faster, more timely resolutions.
+						Note: please read our <a href="https://docs.wpvip.com/technical-references/vip-support/" target="_blank" rel="noopener">ticket guidelines</a>
+						for faster, timely resolutions.
```

Comment:
> faster, timely

"timely" suggests it comes early, or at the right time. But that seems redundant, in so much as someone who opens the tickets needed the answer 5 minutes ago, else they wouldn't have opened the ticket.

I think "faster resolution" is sufficient here since reading and following the guidelines to provide the required information from the beginning, will mean fewer backwards and forwards.

```suggestion
						for faster resolutions.
```